src/hotspot/share/opto/superword.cpp

```diff
@@ -1194,17 +1172,11 @@ bool SuperWord::follow_def_uses(Node_List* p) {
       }
     }
   }
-  if (num_s1_uses > 1) {
-    _race_possible = true;
-  }
```

Comment:
Note: I removed `_race_possible` and `num_s1_uses`. We checked here if there was any node in a pack that has multiple uses. If that happens, it it spossible that `order_inputs_of_uses_to_match_def_pair` changes the order of inputs. This flag was set to fix everything again afterwards. But now I just call that algorithm always, implicitly always setting `_race_possible = true`. This does not cost so much, and makes things quite a bit simpler.